modules/coverage-report/src/test/java/org/jooby/issues/Issue498.java

```diff
@@ -6,7 +6,9 @@
 import org.jooby.test.ServerFeature;
 import org.junit.Test;
 
-import com.couchbase.client.deps.io.netty.util.internal.chmv8.ForkJoinPool;
+import java.util.concurrent.ForkJoinPool;
+
+//import com.couchbase.client.deps.io.netty.util.internal.chmv8.ForkJoinPool;
```

Comment:
remove the commented line

Reply:
Sure, it’s just the output of versions plugin basically. Just leave thin on hold, until i‘m done. I will compile a release note then. About 20-30 more to come, but a full testing cycle needs some time ;-)